Data/UnitTests/GameEngineTest/TypeScript/DataDirectories.ddl

```diff
@@ -22,3 +22,9 @@ DataDir
 	string %RootName{""}
 	bool %Writable{false}
 }
+DataDir
+{
+	string %Path{">sdk/Data/Content"}
+	string %RootName{""}
+	bool %Writable{false}
+}
```

Comment:
These projects don't use the Content data dir, do they? By adding the data directory, the asset transform for these projects may take longer (I guess all assets will only be transformed once, but the redundant projects will still scan the additional assets for changes).

Reply:
They all do use the >sdk/Data/UnitTests/GameEngineTest/SharedAssets data dir which takes all its textures for its materials from the >sdk/Data/Content directory. Removing that dependency looks a bit more difficult without splitting the data dir in two.

Reply:
That's fine then, wasn't aware of that.